tests/test_main.cpp

```diff
@@ -122,6 +122,7 @@
 #include "tests/scene/test_image_texture_3d.h"
 #include "tests/scene/test_instance_placeholder.h"
 #include "tests/scene/test_node.h"
+#include "tests/scene/test_occluder_instance_3d.h"
```

Comment:
Needs to be moved down to the `_3D_DISABLED` section

Reply:
Done!